ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly-4.17-upgrade-from-stable-4.14.yaml

```diff
@@ -33,72 +33,98 @@ build_root:
     namespace: openshift
     tag: golang-1.18
 releases:
-  intermediate413:
-    candidate:
-      architecture: arm64
-      product: ocp
-      stream: nightly
-      version: "4.13"
-  intermediate414:
+  intermediate415:
     candidate:
       architecture: arm64
       product: ocp
       stream: nightly
-      version: "4.14"
-  intermediate415:
+      version: "4.15"
+  intermediate416:
     candidate:
       architecture: arm64
       product: ocp
       stream: nightly
-      version: "4.15"
+      version: "4.16"
   latest:
     release:
       architecture: arm64
       channel: fast
-      version: "4.12"
+      version: "4.14"
   target:
     candidate:
       architecture: arm64
       product: ocp
       stream: nightly
-      version: "4.16"
+      version: "4.17"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-ovn-ipsec-f14
-  cron: 23 19 10,24 * *
+- as: aws-ipi-ovn-ipsec-to-multiarch-f28
```

Comment:
`to-multiarch` jobs are used to migrate sigle-arch cluster to mult-arch cluster, its purpose is not for chain upgrade,  so I do not think they are supposed to be shown here. cc @shellyyang1989 to confirm.

Reply:
I think you are right, I will choose another `ipsec` profile, how about `ibmcloud-ipi-ovn-ipsec`?

Reply:
Replaced with ibmcloud-ipi-ovn-ipsec